tests/src/e2e/ledger_tests.rs

```diff
@@ -4432,11 +4433,15 @@ fn test_epoch_sleep() -> Result<()> {
     // 3. Use epoch-sleep to sleep for an epoch
     let args = ["epoch-sleep", "--node", &validator_one_rpc];
     let mut client = run!(test, Bin::Client, &args, None)?;
+    let reached_epoch = parse_reached_epoch(&mut client)?;
     client.assert_success();
 
     // 4. Confirm the current epoch is larger
+    // possibly badly, we assume we get here within 30 seconds of the last step
+    // should be fine haha (future debuggers: sorry)
```

Comment:
I think 30 secs should be plenty even with very limited resources, but we don't have to add the check for equality